### PR TITLE
[moleculer-mail] Upgrade to email-templates@5

### PR DESCRIPTION
**Pull request by Embraser01**
This PR update `email-templates` to v5 (and every other dependencies of moleculer-mail).

It comes with a few **breaking changes** detailed in the CHANGELOG.md file

So I propose bumping the package to `2.0.0`.

I'd like your review on this changes as I had to make arbitrary decisions and I'm not sure about all of them :smile:

### CODE REVIEW

**Comment by icebob** [comment]
Thank you for your PR. I need more time to check it.

File: packages/moleculer-mail/test/unit/template.spec.js [hunk 1]
```diff
@@ -1,186 +0,0 @@
-"use strict";
-
-const { ServiceBroker } = require("moleculer");
-const MailService = require("../../src");
-const _ = require("lodash");
-
-describe("Test MailService template handling", () => {
-
```
Comment (icebob) [inline packages/moleculer-mail/test/unit/template.spec.js]: Why did you remove these files?

Comment (Embraser01) [reply packages/moleculer-mail/test/unit/template.spec.js]: I removed this file because it was testing the `getTemplate()` method which I removed.

Also, the rendering is not made by us anymore but by `email-templates` so testing the rendered email was not our concern anymore (IMO).

If you think of tests I missed or wrongfully removed, I'd be happy to write them back :smile:

**Comment by Embraser01** [comment]
I just updated dependencies to the latest version (and removed nodemailer-html-to-text dependency)

**Comment by icebob** [comment]
@Embraser01, unfortunately, email-templates goes in a very different way than the early version (v2) what I have implemented. It means, I won't upgrade it, instead, I will rewrite the moleculer-mail without it. Sorry.

**Comment by Embraser01** [comment]
> @Embraser01, unfortunately, email-templates goes in a very different way than the early version (v2) what I have implemented. It means, I won't upgrade it, instead, I will rewrite the moleculer-mail without it. Sorry.

I understand, no problem 😃
